core/domain/exp_domain.py

```diff
@@ -232,7 +232,7 @@ def __init__(
                 made in this model, which should give sufficient information to
                 reconstruct the commit. Each dict always contains the following
                 key:
-                    - cmd: str. Unique command.
+                - cmd (str). Unique command.
```

Comment:
I think it was correct before, actually -- can you keep the formatting of "argname: typeinfo."?

Reply:
So this is the part where I don't know how to handle this case. Since key isn't an argname parameter, it's part of the description for the above parameter, because of that it messes up my code.

Reply:
Done